conf/crail-env.sh.template

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
 # This env varibale allows setting additional java parameter
 #CRAIL_EXTRA_JAVA_OPTIONS="-Xmx24G -Xmn16G"
+
+bin=`dirname "${BASH_SOURCE-$0}"`
+bin=`cd "$bin"; pwd`
+export CRAIL_HOME="$bin"/..
```

Comment:
I'm ok with putting CRAIL_HOME in the env file however we don't want to bother the user with the bash vodoo to get the path from the BASH_SOURCE. Again I think the same can be achieve by putting this logic into crail-deamon.sh with a check if CRAIL_HOME has been set. If it hasn't we fall back to the code above. Currently the start-crail.sh script also uses this logic thus it would be nice if we could combine this somewhere.